tsconfig.json

```diff
@@ -19,10 +19,10 @@
       "dom"
     ],
     "paths": {
-      "angular-image-viewer": [
+      "@clarivate/angular-image-viewer": [
```

Comment:
Still its not reflecting in UI if I change anything in component file.

I started "yarn serve-lib" then "yarn start" but any changes, it compile and push it to below path as per "projects\angular-image-viewer\ng-package.json" file "dest": "../../dist/angular-image-viewer",

UI refreshes but changes not reflecting because it takes always from node_module/@clarivate/angular-image-viewer.

For my development work, i changed to node_module path instead of dist in ng-package.json and restart server. I added the same in Readme file.

Reply:
this seems ok to me when i ran it...